### ISSUE TITLE
[bug]: Radio buttons in arguments table of docs have an invalid ARIA attribute

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5254]
### 🐞 Describe the bug

Radio buttons in arguments table of the documentation are wrapped in a div that has an `aria-readonly="false"` attribute.
As stated in the [mdn web docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-readonly), `aria-readonly` should be used in editable content. Either remove the attribute or add a `role="radiogroup"` to the div to make it valid.

**Comment by leagrdv** [comment]
The issue is within storybook and is not fixable with just config. I've opened a ticket for it on their repo: [Link to the ticket](https://github.com/storybookjs/storybook/issues/31436)
